src/server.c

```diff
@@ -3594,8 +3594,8 @@ int commandCheckArity(client *c, sds *err) {
  * if C_ERR is returned the client was destroyed (i.e. after QUIT). */
 int processCommand(client *c) {
     if (!scriptIsTimedout()) {
-        /* Both EXEC and EVAL call call() directly so there should be
-         * no way in_exec or in_eval is 1.
+        /* Both EXEC and script call call() directly so there should be
+         * no way in_exec or scriptIsRunning() is 1.
```

Comment:
We are not using `in_eval` anymore, it was replaced by in_script and then by scriptIsRunning in #10725